docs/configuring-openshift.md

```diff
@@ -0,0 +1,69 @@
+# Configuring OpenShift
+The following sections describe how Minishift can be used to configure your local OpenShift instance provisioned by the Minishift VM. For details about managing the Minishift VM refer to the [Using Minishift](./using.md) section.
```

Comment:
"The following sections describe how Minishift can assist in the configuration of ..."